torch/_dynamo/symbolic_convert.py

```diff
@@ -471,6 +471,18 @@ def call_function(
             isinstance(x, VariableTracker)
             for x in itertools.chain(args, kwargs.values())
         )
+        inner_fn = None
+        if hasattr(fn, "value"):
+            inner_fn = fn.value
+        if hasattr(fn, "fn"):
+            inner_fn = fn.fn
```

Comment:
This is very suspicious. Why does VariableTracker not have a way to generally query attributes in this way...

Reply:
Legacy code, I could put up a refactor PR ahead of this to make `.value` the way.